### DISCUSSION TITLE
Is this the worst time to install OS 7?

### DISCUSSION THREAD

**Discussion by lo-yak** [elementary/os#719] [Q&A] [answered]
Hey!
I want to install elementary, but... Since OS 8 is close to release and since apparently I'll have to reinstall the system to upgrade, I'm a bit stuck because of the lack of a specific date for release.

I totally get and appreciate the "we ship when we're ready" approach, but when this goes together with the inability to upgrade without reinstalling, then it becomes a bit of a dilemma. I mean, I could install OS 7 today and have to reinstall it in one week, or 

I could decide to install OS 8 directly and have to wait for months.

I don't know, I hope this makes sense. I just wanted to know if I'm missing something or maybe you have any suggestion. 
Thanks!

**Comment by danirabbit** [comment, marked as answer]
We're definitely not only a week away from OS 8 release. I'm optimistically hoping for maybe end of September release. But realistically we can't release until [this project board](https://github.com/orgs/elementary/projects/128) is cleared and there's no real way to make a promise about how long that will take. Sorry I know that's not definitive, but I hope it's helpful!

For what it's worth, OS 7 continues to receive security and stability updates until 2027, we're still making limited bug fix and feature updates to OS 7 when it's simply to back port them from OS 8, and Flatpak apps (Ones we ship pre-installed, ones from AppCenter, and ones from alt stores like Flathub) will continue to receive all updates indefinitely. So there's no rush to upgrade as soon as OS 8 is released :)

**Reply by lo-yak** [reply]
Oh, that's great! I'll install 7 then, and jump to 8 with no rush. I actually could benefit from the inability to upgrade without reinstalling, it should help with keeping the system clean, and that's appreciated lol

Thank you Danielle :)
